lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -0,0 +1,246 @@
+        if sid:
+            actions.append(parser.OFPActionSetField(metadata=sid))
+
+        flows.add_flow(self._datapath, self.tbl_num, match, actions=actions,
+                       priority=priority, goto_table=self.next_table)
+
+        # Add flow for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+                           in_port=self.config.mtr_port,
+                           ipv4_dst=ue_ip_adr)
+
+        actions = [parser.OFPActionSetField(tunnel_id=o_teid),
+                   parser.OFPActionSetField(tun_ipv4_dst=enodeb_ip_addr)]
+        if sid:
+            actions.append(parser.OFPActionSetField(metadata=sid))
+
+        flows.add_flow(self._datapath, self.tbl_num, match, actions=actions,
+                       priority=priority, goto_table=self.next_table)
+       
+        # Add ARP flow for LOCAL port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_ARP,
+                           in_port=self._uplink_port, arp_tpa=ue_ip_adr)
+        if sid:
+            actions = [parser.OFPActionSetField(metadata=sid)]
+
+        flows.add_flow(self._datapath, self.tbl_num, match, actions=actions,
+                       priority=priority, goto_table=self.next_table)
+
+        # Add ARP flow for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_ARP,
+                               in_port=self.config.mtr_port,
+                               arp_tpa=ue_ip_adr)
+        if sid:
+            actions = [parser.OFPActionSetField(metadata=sid)]
+
+        flows.add_flow(self._datapath, self.tbl_num, match, actions=actions,
+                       priority=priority, goto_table=self.next_table)
+
+
+    def _delete_tunnel_flows(self, i_teid:int, ue_ip_adr:str):
+
+        # Delete flow for gtp port
+        match = MagmaMatch(tunnel_id=i_teid, in_port=self.config.gtp_port)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete flow for LOCAL port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+                           in_port=self._uplink_port, ipv4_dst=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete flow for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+                           in_port=self.config.mtr_port,ipv4_dst=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete ARP flow for LOCAL port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_ARP,
+                           in_port=self._uplink_port, arp_tpa=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+        # Delete ARP flow for mtr port
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_ARP,
+                           in_port=self.config.mtr_port, arp_tpa=ue_ip_adr)
+
+        flows.delete_flow(self._datapath, self.tbl_num, match)
+
+
+    def _add_discard_data_tunnel_flows(self, precedence:int, i_teid:int,
```

Comment:
This function is used for CSFB type of support. We should not be needing this particular function moving forward (e.g., if for whatever reason we need to drop the flows for a UE and not page it until some control plane trigger happens, it is better to treat this as an inactive state with no paging). @pshelar and @ssanadhya what do you think?